### ISSUE TITLE
Error when table field is passed through integration and is null

### ISSUE THREAD

**Issue by frankkepler** [verbb/formie#935]
**Description**
Error when a table field is saved with a null value and passed through an integration to a craft table field.

`[error][formie] Element integration failed for submission “8599”. Error: array_key_exists(): Argument #2 ($array) must be of type array, null given /Users/name/folder/company/vendor/craftcms/cms/src/fields/Table.php:396`

**Steps to reproduce**
1. Create a form with a table field 
2. Enable an integration on the form that connects the table field to a corresponding table field in Craft
3. Submit the form with the table field empty (null)

### Form settings
Please pick from the options below.

- Multi-page form: Yes
- Submission Method: Page Reload
- Client-side Validation: Yes 
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 2.7.2
- Craft version: 3.7.44
- Multi-site: No

**Additional context**

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.19",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.6.0](https://github.com/verbb/formie/releases/tag/1.6.0)
